src/renderer/protocol-handler/app-handlers.ts

```diff
@@ -16,18 +15,10 @@ export function bindProtocolAddRouteHandlers() {
       navigate(preferencesURL({ fragment: highlight }));
     })
     .addInternalHandler("/", () => {
-      navigate(landingURL());
+      navigate(catalogURL());
     })
-    .addInternalHandler("/landing", () => {
-      navigate(landingURL());
-    })
-    .addInternalHandler("/landing/:workspaceId", ({ pathname: { workspaceId } }) => {
-      if (workspaceStore.getById(workspaceId)) {
-        workspaceStore.setActive(workspaceId);
-        navigate(landingURL());
-      } else {
-        console.log("[APP-HANDLER]: workspace with given ID does not exist", { workspaceId });
-      }
+    .addInternalHandler("/catalog", () => {
+      navigate(catalogURL());
```

Comment:
I still think we should keep `/landing` for backward comparability purposes. Plus it is the landing page, so we don't really need `/catalog`

Reply:
You are right, I somehow missed that we also have `/` for this. I'll switch back to `/landing` for backward compat purposes.